src/main/java/com/university/university/controller/UniversityController.java

```diff
@@ -1,5 +1,7 @@
 package com.university.university.controller;

```

Comment:
please refactor package name to 
package com.agilethought.internship.university.controller;